lib/packages.cf

```diff
@@ -32,13 +32,20 @@ bundle common package_module_knowledge
     windows::
       "platform_default" string => "msiexec";
 @endif
+
+    termux::
+      "platform_default" string => "apt_get";
 }
 
 body package_module apt_get
 {
     query_installed_ifelapsed => "$(def.package_module_query_installed_ifelapsed)";
     query_updates_ifelapsed => "$(def.package_module_query_updates_ifelapsed)";
     #default_options =>  {};
+@if minimum_version(3.15.1)
```

Comment:
Perhaps hash comment about why this is targeting 3.15.1 instead of 3.12.something (because it woudlnt work for 3.13.